### DISCUSSION TITLE
How to enable smooth scroll?

### DISCUSSION THREAD

**Discussion by tiziodcaio** [vuepress/core#206] [Q&A] [answered]
There isn't in the documentations... so I will ask here

**Comment by meteorlxy** [comment, marked as answer]
You can also ask Google.

https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior

**Reply by tiziodcaio** [reply]
Super I will try

**Reply by tiziodcaio** [reply]
But.... How about old smooth scroll of vuepress?

**Reply by Mister-Hope** [reply]
The old smooth scroll is using `smooth-scroll` to pollyfill old browsers in my memorys.

But vuepress2 do not supprt old browsers like ie, so most of the browsers don't need to polyfill

**Reply by Mister-Hope** [reply]
```css
:root{
 scroll-behavior: smooth
}
```

should be enough for most browsers expect safari

![image](https://user-images.githubusercontent.com/33315834/121713903-ee6cfc80-cb0f-11eb-8c83-eab2a114d931.png)

**Reply by tiziodcaio** [reply]
Thanks
